### PR TITLE
LOTR 17 cards

### PR DESCRIPTION
**Pull request by Simisays**

### CODE REVIEW

File: forge-gui/res/cardsfolder/upcoming/The_grey_havens.txt [hunk 1]
```diff
@@ -0,0 +1,9 @@
+Name:The Grey Havens
+ManaCost:no cost
+Types:Legendary Land
+T:Mode$ ChangesZone | Origin$ Any | Destination$ Battlefield | ValidCard$ Card.Self | Execute$ TrigScry | TriggerDescription$ When CARDNAME enters the battlefield, scry 1.
+SVar:TrigScry:DB$ Scry | ScryNum$ 1
+A:AB$ Mana | Cost$ T | Produced$ C | Amount$ 1 | SpellDescription$ Add {C}
+A:AB$ ManaReflected | Cost$ T | ColorOrType$ Color | Valid$ Defined.ValidGraveyard Creature.Legendary+YouOwn | ReflectProperty$ Is | SpellDescription$ Add one mana of any color among legendary creature cards in your graveyard.
+DeckHints:Type$Legendary
```
Comment (tool4ever) [inline forge-gui/res/cardsfolder/upcoming/The_grey_havens.txt:8]: `& Type$Creature`

File: forge-gui/res/cardsfolder/upcoming/saruman_of_many_colors.txt [hunk 2]
```diff
@@ -0,0 +1,14 @@
+Name:Saruman of Many Colors
+ManaCost:3 W U B
+Types:Legendary Creature Avatar Wizard
+PT:5/4
+K:Ward:Discard<1/Enchantment;Instant;Sorcery>
+T:Mode$ SpellCast | ValidCard$ Card.YouCtrl | ValidActivatingPlayer$ You | TriggerZones$ Battlefield | Execute$ TrigMill | ActivatorThisTurnCast$ EQ2 | TriggerDescription$ Whenever you cast your second spell each turn, each opponent mills two cards. When one or more cards are milled this way, exile target enchantment, instant, or sorcery card with equal or lesser mana value than that spell from an opponent's graveyard. Copy the exiled card. You may cast the copy without paying its mana cost.
+SVar:TrigMill:DB$ Mill | Defined$ Opponent | NumCards$ 2 | RememberMilled$ True | SubAbility$ DBExile
```
Comment (tool4ever) [inline forge-gui/res/cardsfolder/upcoming/saruman_of_many_colors.txt:7]: Use release notes:
> You don't choose a target for Saruman of Many Colors's ability at the time it triggers. Rather, a second "reflexive" ability triggers when one or more cards are milled this way. You choose a target for that ability as it goes on the stack. Each player may respond to this triggered ability as normal.

File: forge-gui/res/cardsfolder/upcoming/shelobs_ambush.txt [hunk 3]
```diff
@@ -0,0 +1,7 @@
+Name:Shelob's Ambush
+ManaCost:B
+Types:Instant
+A:SP$ Pump | ValidTgts$ Creature | KW$ Deathtouch | AddPower$ 1 | AddToughness$ 2 | SubAbility$ DBFood | SpellDescription$ Target creature gets +1/+2 and gains deathtouch until end of turn.
+SVar:DBFood:DB$ Token | TokenAmount$ 1 | TokenScript$ c_a_food_sac | SpellDescription$  Create a Food token. (It's an artifact with "{2}, {T}, Sacrifice this artifact: You gain 3 life.")
+DeckHas:Ability$Food|LifeGain & Type$Food|Artifact
```
Comment (tool4ever) [inline forge-gui/res/cardsfolder/upcoming/shelobs_ambush.txt:6]: I don't think Food is Ability

File: forge-gui/res/cardsfolder/upcoming/smite_the_deathless.txt [hunk 4]
```diff
@@ -0,0 +1,8 @@
+Name:Smite the Deathless
+ManaCost:1 R
+Types:Instant
+A:SP$ DealDamage | ValidTgts$ Creature | NumDmg$ 3 | RememberDamaged$ True | ReplaceDyingDefined$ Remembered.Creature | SubAbility$ DBDebuff | SpellDescription$ CARDNAME deals 3 damage to target creature. That creature loses indestructible until end of turn. If that creature would die this turn, exile it instead.
```
Comment (tool4ever) [inline forge-gui/res/cardsfolder/upcoming/smite_the_deathless.txt:4]: work with Targeted

File: forge-gui/res/cardsfolder/upcoming/smite_the_deathless.txt [hunk 5]
```diff
@@ -0,0 +1,8 @@
...
+SVar:DBDebuff:DB$ Debuff | Defined$ Remembered | Keywords$ Indestructible | SubAbility$ DBCleanup
+SVar:DBCleanup:DB$ Cleanup | ClearRemembered$ True
+SVar:X:Count$xPaid
```
Comment (tool4ever) [inline forge-gui/res/cardsfolder/upcoming/smite_the_deathless.txt:7]: ?

File: forge-gui/res/cardsfolder/upcoming/spiteful_banditry.txt [hunk 6]
```diff
@@ -0,0 +1,10 @@
+Name:Spiteful Banditry
+ManaCost:X R R
+Types:Enchantment
+T:Mode$ ChangesZone | Origin$ Any | Destination$ Battlefield | ValidCard$ Card.Self | Execute$ TrigDamageAll | TriggerDescription$ When CARDNAME enters the battlefield, it deals X damage to each creature.
+SVar:TrigDamageAll:DB$ DamageAll | NumDmg$ X | ValidCards$ Creature
+SVar:X:Count$xPaid
+T:Mode$ ChangesZoneAll | ValidCards$ Creature.OppCtrl | ActivationLimit$ 1 | Origin$ Battlefield | Destination$ Graveyard | TriggerZones$ Battlefield | ActivationLimit$ 1 | Execute$ TrigTreasure | TriggerDescription$ Whenever one or more creatures your opponents control die, you create a Treasure token. This ability triggers only once each turn.
+SVar:TrigTreasure:DB$ Token | TokenAmount$ 1 | TokenScript$ c_a_treasure_sac | TokenOwner$ You 
+Oracle:When Spiteful Banditry enters the battlefield, it deals X damage to each creature.
+Whenever one or more creatures your opponents control die, you create a Treasure token. This ability triggers only once each turn.
```
Comment (tool4ever) [inline forge-gui/res/cardsfolder/upcoming/spiteful_banditry.txt:10]: wrong break

File: forge-gui/res/cardsfolder/upcoming/stern_scolding.txt [hunk 7]
```diff
@@ -0,0 +1,5 @@
+Name:Stern Scolding
+ManaCost:U
+Types:Instant
+A:SP$ Counter | Cost$ 1 U | TargetType$ Spell | TgtPrompt$ Select target Creature spell | ValidTgts$ Creature.powerLE2,Creature.Toughness.LE2 | TgtPrompt$ Select target creature spell with power or toughness 2 or less | SpellDescription$ Counter target creature spell with power or toughness 2 or less.
```
Comment (tool4ever) [inline forge-gui/res/cardsfolder/upcoming/stern_scolding.txt:4]: fix Tgts

File: forge-gui/res/cardsfolder/upcoming/stew_the_coneys.txt [hunk 8]
```diff
@@ -0,0 +1,9 @@
+Name:Stew the Coneys
+ManaCost:2 W
+Types:Instant
+A:SP$ Pump | ValidTgts$ Creature.YouCtrl | AILogic$ PowerDmg | TgtPrompt$ Select target creature you control | SubAbility$ SoulsDamage | StackDescription$ None | SpellDescription$ Target creature you control deals damage equal to its power to target creature you don't control
+SVar:SoulsDamage:DB$ DealDamage | ValidTgts$ Creature.YouDontCtrl | AILogic$ PowerDmg | TgtPrompt$ Select target creature you don't control | NumDmg$ X | DamageSource$ ParentTarget | SubAbility$ DBFood
+SVar:DBFood:DB$ Token | TokenScript$ c_a_food_sac | TokenOwner$ You | TokenAmount$ X | SpellDescription$ Create a Food token. (It's an artifact with "{2}, {T}, Sacrifice this artifact: You gain 3 life.")
+SVar:X:ParentTargeted$CardPower
```
Comment (tool4ever) [inline forge-gui/res/cardsfolder/upcoming/stew_the_coneys.txt:7]: ?

File: forge-gui/res/cardsfolder/upcoming/stew_the_coneys.txt [hunk 9]
```diff
@@ -0,0 +1,9 @@
+Name:Stew the Coneys
+ManaCost:2 W
+Types:Instant
+A:SP$ Pump | ValidTgts$ Creature.YouCtrl | AILogic$ PowerDmg | TgtPrompt$ Select target creature you control | SubAbility$ SoulsDamage | StackDescription$ None | SpellDescription$ Target creature you control deals damage equal to its power to target creature you don't control
+SVar:SoulsDamage:DB$ DealDamage | ValidTgts$ Creature.YouDontCtrl | AILogic$ PowerDmg | TgtPrompt$ Select target creature you don't control | NumDmg$ X | DamageSource$ ParentTarget | SubAbility$ DBFood
+SVar:DBFood:DB$ Token | TokenScript$ c_a_food_sac | TokenOwner$ You | TokenAmount$ X | SpellDescription$ Create a Food token. (It's an artifact with "{2}, {T}, Sacrifice this artifact: You gain 3 life.")
```
Comment (tool4ever) [inline forge-gui/res/cardsfolder/upcoming/stew_the_coneys.txt:6]: wrong amount

File: forge-gui/res/cardsfolder/upcoming/saruman_of_many_colors.txt (hunk 2 above)
Comment (Simisays) [reply forge-gui/res/cardsfolder/upcoming/saruman_of_many_colors.txt]: I think with the immediatetrigger the `SVar:X:TriggeredCard$CardManaCost` doesn't work anymore ?

File: forge-gui/res/cardsfolder/upcoming/saruman_of_many_colors.txt [hunk 10]
```diff
@@ -0,0 +1,15 @@
...
+ManaCost:3 W U B
+Types:Legendary Creature Avatar Wizard
+PT:5/4
+K:Ward:Discard<1/Enchantment;Instant;Sorcery>
+T:Mode$ SpellCast | ValidCard$ Card.YouCtrl | ValidActivatingPlayer$ You | TriggerZones$ Battlefield | Execute$ TrigMill | ActivatorThisTurnCast$ EQ2 | TriggerDescription$ Whenever you cast your second spell each turn, each opponent mills two cards. When one or more cards are milled this way, exile target enchantment, instant, or sorcery card with equal or lesser mana value than that spell from an opponent's graveyard. Copy the exiled card. You may cast the copy without paying its mana cost.
+SVar:TrigMill:DB$ Mill | Defined$ Opponent | NumCards$ 2 | RememberMilled$ True | SubAbility$ DBImmediateTrigger
+SVar:DBImmediateTrigger:DB$ ImmediateTrigger | ConditionDefined$ Remembered | ConditionPresent$ Card | RememberObjects$ Remembered | ConditionCompare$ GE1 | Execute$ TrigExile | SubAbility$ DBCleanup | TriggerDescription$ When one or more cards are milled this way, exile target enchantment, instant, or sorcery card with equal or lesser mana value than that spell from an opponent's graveyard. Copy the exiled card. You may cast the copy without paying its mana cost.
```
Comment (tool4ever) [inline forge-gui/res/cardsfolder/upcoming/saruman_of_many_colors.txt:8]: doesn't make sense to remember those
use `CopyTriggeringObjects$`

File: forge-gui/res/cardsfolder/upcoming/saruman_of_many_colors.txt [hunk 11]
```diff
@@ -0,0 +1,15 @@
...
+SVar:TrigExile:DB$ ChangeZone | Origin$ Graveyard | Destination$ Exile | TgtPrompt$ Choose target instant or sorcery card in a graveyard | ValidTgts$ Instant.OppOwn+cmcLEX,Sorcery.OppOwn+cmcLEX  | TgtPrompt$ Select target instant or sorcery card from an opponent's graveyard | Imprint$ True | SubAbility$ DBPlay
+SVar:DBPlay:DB$ Play | Valid$ Card.IsImprinted | ValidZone$ Exile | Controller$ You | CopyCard$ True | WithoutManaCost$ True | ValidSA$ Spell | Optional$ True | Amount$ All
```
Comment (tool4ever) [inline forge-gui/res/cardsfolder/upcoming/saruman_of_many_colors.txt:10]: doesn't this leave imprinted behind?

File: forge-gui/res/cardsfolder/upcoming/smite_the_deathless.txt (hunk 4 above)
Comment (tool4ever) [reply forge-gui/res/cardsfolder/upcoming/smite_the_deathless.txt]: Defined too

File: forge-gui/res/cardsfolder/upcoming/shire_terrace.txt [hunk 12]
```diff
@@ -0,0 +1,7 @@
+Name:Shire Terrace
+ManaCost:no cost
+Types:Land
+A:AB$ Mana | Cost$ T | Produced$ C | SpellDescription$ Add {C}.
+A:AB$ ChangeZone | Cost$ 1 T Sac<1/CARDNAME> | Origin$ Library | Destination$ Battlefield | Tapped$ True | ChangeType$ Land.Basic | ChangeNum$ 1 | SpellDescription$ Search your library for a basic land card, put it onto the battlefield tapped, then shuffle.
+DeckHas:Ability$Sacrifice
+Oracle:{T}:: Add {C}.\n{1}, {T}, Sacrifice Shire Terrace: Search your library for a basic land card, put it onto the battlefield tapped, then shuffle.
```
Comment (tool4ever) [inline forge-gui/res/cardsfolder/upcoming/shire_terrace.txt:7]: ::

File: forge-gui/res/cardsfolder/upcoming/shower_of_arrows.txt [hunk 13]
```diff
@@ -0,0 +1,7 @@
+Name:Shower of Arrows
+ManaCost:2 G
+Types:Instant
+A:SP$ Destroy | ValidTgts$ Artifact,Battle,Enchantment,Creature.withFlying | SubAbility$ DBScry | TgtPrompt$ Select target artifact, enchantment, or creature with flying | SpellDescription$ Destroy target artifact, enchantment, or creature with flying.
```
Comment (tool4ever) [inline forge-gui/res/cardsfolder/upcoming/shower_of_arrows.txt:4]: Why Battle...

File: forge-gui/res/cardsfolder/upcoming/stern_scolding.txt [hunk 14]
```diff
@@ -0,0 +1,5 @@
+Name:Stern Scolding
+ManaCost:U
+Types:Instant
+A:SP$ Counter | Cost$ 1 U | TargetType$ Spell | TgtPrompt$ Select target Creature spell | ValidTgts$ Creature.powerLE2,Creature.toughnessLE2 | TgtPrompt$ Select target creature spell with power or toughness 2 or less | SpellDescription$ Counter target creature spell with power or toughness 2 or less.
```
Comment (tool4ever) [inline forge-gui/res/cardsfolder/upcoming/stern_scolding.txt:4]: TgtPrompt twice

File: forge-gui/res/cardsfolder/upcoming/stone_of_erech.txt [hunk 15]
```diff
@@ -0,0 +1,9 @@
+Name:Stone of Erech
+ManaCost:1
+Types:Legendary Artifact
+R:Event$ Moved | ActiveZones$ Battlefield | Origin$ Battlefield | Destination$ Graveyard | ValidCard$ Creature.OppOwn | ReplaceWith$ Exile | Description$ If a creature an opponent controls would die, exile it instead.
```
Comment (tool4ever) [inline forge-gui/res/cardsfolder/upcoming/stone_of_erech.txt:4]: wrong ValidCard 😩

File: forge-gui/res/cardsfolder/upcoming/friendly_rivalry.txt [hunk 16]
```diff
@@ -0,0 +1,7 @@
+Name:Friendly Rivalry
+ManaCost:R G
+Types:Instant
+A:SP$ Pump | ValidTgts$ Creature.YouCtrl | TgtPrompt$ Select target creature you control | SubAbility$ DBPump | SpellDescription$ Target creature you control and up to one other target legendary creature you control each deal damage equal to their power to target creature you don't control.
+SVar:DBPump:DB$ Pump | ValidTgts$ Creature.Legendary+YouCtrl | TgtPrompt$ Select up to one target legendary creature you control | TargetMin$ 0 | TargetMax$ 1 | TargetUnique$ True | SubAbility$ DBEachDamage 
+SVar:DBEachDamage:DB$ EachDamage | ValidTgts$ Creature.YouDontCtrl | TgtPrompt$ Select target creature you don't control | DefinedDamagers$ ParentTarget | NumDmg$ Count$CardPower
```
Comment (tool4ever) [inline forge-gui/res/cardsfolder/upcoming/friendly_rivalry.txt:6]: I don't think this will grab both damagers

File: forge-gui/res/cardsfolder/upcoming/the_balrog_durins_bane.txt [hunk 17]
```diff
@@ -0,0 +1,12 @@
+Name:The Balrog, Durin's Bane
+ManaCost:5 B R
+Types:Legendary Creature Avatar Demon
+PT:7/5
+S:Mode$ ReduceCost | ValidCard$ Card.Self | Type$ Spell | Amount$ X | Relative$ True | EffectZone$ All | Description$ This spell costs {1} less to cast for each permanent sacrificed this turn.
+SVar:X:Count$SacrificedThisTurn Permanent
```
Comment (tool4ever) [inline forge-gui/res/cardsfolder/upcoming/the_balrog_durins_bane.txt:6]: outdated call

File: forge-gui/res/cardsfolder/upcoming/friendly_rivalry.txt (hunk 16 above)
Comment (tool4ever) [reply forge-gui/res/cardsfolder/upcoming/friendly_rivalry.txt]: now the third would probably also damage itself...

File: forge-gui/res/cardsfolder/upcoming/The_grey_havens.txt
Comment (tool4ever) [inline forge-gui/res/cardsfolder/upcoming/The_grey_havens.txt:1]: caps

File: forge-gui/res/cardsfolder/upcoming/the_balrog_durins_bane.txt (hunk 17 above)
Comment (Simisays) [reply forge-gui/res/cardsfolder/upcoming/the_balrog_durins_bane.txt]: Is this still wrong ? I've used Pyre-Sledge Arsonist as a template

Comment (tool4ever) [reply forge-gui/res/cardsfolder/upcoming/the_balrog_durins_bane.txt]: Yes syntax wrong slightly but logic would be wrong too because you ignore allies
Just use `PlayerCount$` 🤷‍♂️

Comment (tool4ever) [reply forge-gui/res/cardsfolder/upcoming/the_balrog_durins_bane.txt]: Also the `Relative$` doesn't make sense

File: forge-gui/res/cardsfolder/upcoming/saruman_of_many_colors.txt [hunk 18]
```diff
@@ -0,0 +1,16 @@
...
+Types:Legendary Creature Avatar Wizard
+PT:5/4
+K:Ward:Discard<1/Enchantment;Instant;Sorcery>
+T:Mode$ SpellCast | ValidCard$ Card.YouCtrl | ValidActivatingPlayer$ You | TriggerZones$ Battlefield | Execute$ TrigMill | ActivatorThisTurnCast$ EQ2 | TriggerDescription$ Whenever you cast your second spell each turn, each opponent mills two cards. When one or more cards are milled this way, exile target enchantment, instant, or sorcery card with equal or lesser mana value than that spell from an opponent's graveyard. Copy the exiled card. You may cast the copy without paying its mana cost.
+SVar:TrigMill:DB$ Mill | Defined$ Opponent | NumCards$ 2 | RememberMilled$ True | SubAbility$ DBImmediateTrigger
+SVar:DBImmediateTrigger:DB$ ImmediateTrigger | ConditionDefined$ Remembered | ConditionPresent$ Card | RememberObjects$ Remembered | CopyTriggeringObjects$ True | Execute$ TrigExile | SubAbility$ DBCleanup | TriggerDescription$ When one or more cards are milled this way, exile target enchantment, instant, or sorcery card with equal or lesser mana value than that spell from an opponent's graveyard. Copy the exiled card. You may cast the copy without paying its mana cost.
```
Comment (tool4ever) [inline forge-gui/res/cardsfolder/upcoming/saruman_of_many_colors.txt:8]: why did you add RememberObjects again?

Comment (Simisays) [reply forge-gui/res/cardsfolder/upcoming/saruman_of_many_colors.txt:8]: Was trying stuff since it's currently not working (anymore)

File: forge-gui/res/cardsfolder/upcoming/saruman_of_many_colors.txt [hunk 19]
```diff
@@ -0,0 +1,16 @@
+Name:Saruman of Many Colors
+ManaCost:3 W U B
+Types:Legendary Creature Avatar Wizard
+PT:5/4
+K:Ward:Discard<1/Enchantment;Instant;Sorcery>
```
Comment (tool4ever) [inline forge-gui/res/cardsfolder/upcoming/saruman_of_many_colors.txt:5]: Needs TypeDesc

File: forge-gui/res/cardsfolder/upcoming/saruman_of_many_colors.txt (hunk 18 above)
Comment (tool4ever) [reply forge-gui/res/cardsfolder/upcoming/saruman_of_many_colors.txt:8]: ok CopyTriggeringObjects$ doesn't perform good enough, I wanted to get rid of it eventually anyway

So just Remember the TriggeredCard instead

File: forge-gui/res/cardsfolder/upcoming/saruman_of_many_colors.txt [hunk 20]
```diff
@@ -0,0 +1,16 @@
...
+SVar:TrigMill:DB$ Mill | Defined$ Opponent | NumCards$ 2 | RememberMilled$ True | SubAbility$ DBImmediateTrigger
+SVar:DBImmediateTrigger:DB$ ImmediateTrigger | ConditionDefined$ Remembered | ConditionPresent$ Card | RememberObjects$ TriggeredCard | SubAbility$ DBCleanup | TriggerDescription$ When one or more cards are milled this way, exile target enchantment, instant, or sorcery card with equal or lesser mana value than that spell from an opponent's graveyard. Copy the exiled card. You may cast the copy without paying its mana cost.
+SVar:TrigExile:DB$ ChangeZone | Origin$ Graveyard | Destination$ Exile | TgtZone$ Graveyard | ValidTgts$ Instant.OppOwn+cmcLEX,Sorcery.OppOwn+cmcLEX  | TgtPrompt$ Select target instant or sorcery card from an opponent's graveyard | Imprint$ True | SubAbility$ DBPlay
+SVar:DBPlay:DB$ Play | Valid$ Card.IsImprinted | ValidZone$ Exile | Controller$ You | CopyCard$ True | WithoutManaCost$ True | ValidSA$ Spell | Optional$ True | Amount$ All | SubAbility$ DBCleanupBis
+SVar:DBCleanup:DB$ Cleanup | ClearRemembered$ True
+SVar:DBCleanupBis:DB$ Cleanup | ClearImprinted$ True
+SVar:X:Remembered$CardManaCost
```
Comment (tool4ever) [inline forge-gui/res/cardsfolder/upcoming/saruman_of_many_colors.txt:13]: A trigger doesn't have it's own host that's why you need to use TriggerRemembered

Comment (Simisays) [reply forge-gui/res/cardsfolder/upcoming/saruman_of_many_colors.txt:13]: Thanks for all your help but sadly card is still not working 😞

Comment (tool4ever) [reply forge-gui/res/cardsfolder/upcoming/saruman_of_many_colors.txt:13]: That's because you also randomly removed Execute$

Comment (Simisays) [reply forge-gui/res/cardsfolder/upcoming/saruman_of_many_colors.txt:13]: Oh god no wonder it didn't work 🤦

Comment (Simisays) [reply forge-gui/res/cardsfolder/upcoming/saruman_of_many_colors.txt:13]: Finally got it working, thanks again 
image

Comment (Simisays) [reply forge-gui/res/cardsfolder/upcoming/saruman_of_many_colors.txt]: Still something wrong with it ?

Comment (tool4ever) [reply forge-gui/res/cardsfolder/upcoming/saruman_of_many_colors.txt]: Chance is you're not handling LKI correctly yet

Check with X spell that you counter before resolving the trigger

Comment (Simisays) [reply forge-gui/res/cardsfolder/upcoming/saruman_of_many_colors.txt]: It did still allow me to use Sarumans ability. I guess that's not allowed ? Does it need RememberObjects$ TriggeredCardLKICopy ?

Comment (tool4ever) [reply forge-gui/res/cardsfolder/upcoming/saruman_of_many_colors.txt]: Nothing in the ability forbids the game from using LKI to obtain the value
so yea, that change is probably the right direction

File: forge-gui/res/cardsfolder/upcoming/stone_of_erech.txt [hunk 21]
```diff
@@ -0,0 +1,9 @@
+Name:Stone of Erech
+ManaCost:1
+Types:Legendary Artifact
+R:Event$ Moved | ActiveZones$ Battlefield | Origin$ Battlefield | Destination$ Graveyard | ValidCard$ Creature.OppCtrl | ReplaceWith$ Exile | Description$ If a creature an opponent controls would die, exile it instead.
```
Comment (tool4ever) [inline forge-gui/res/cardsfolder/upcoming/stone_of_erech.txt:4]: shouldn't this use LKI?

**Review by tool4ever — APPROVED** [review]
